Assets/Scripts/Source/Game/GameFlowManager.cs

```diff
@@ -1,6 +1,40 @@
 using UnityEngine;
-
+using UnityEngine.SceneManagement;
 public class GameFlowManager : MonoBehaviour 
 {
-    // TODO
+
+    public GameObject combatCanvas;
+    public GameObject combatUI;
+    public GameObject camera;
+
+    public static GameFlowManager instance = null;
+
+    private void Awake()
+    {
+        if (instance == null)
+            instance = this;
+
+        else if (instance != this)
+            Destroy(gameObject);
+
+        DontDestroyOnLoad(gameObject);
+
+        combatCanvas.SetActive(false);
+        combatUI.SetActive(false);
+        camera.SetActive(false);
+
+        SceneManager.LoadScene("Intro", LoadSceneMode.Additive);
+    }
+
+    public void UnloadScene(string sceneName)
+    {
+        SceneManager.UnloadSceneAsync(sceneName);
+
+        if (sceneName == "Intro")
```

Comment:
Nit: each scene could have a list of GameObjects or a delegate handler associated w/ its load/unload